src/js/components/Pagination/Pagination.js

```diff
@@ -176,9 +178,41 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
+    const itemsBegin = step * (activePage - 1) + 1;
+    const itemsEnd = itemsBegin + step - 1;
+    let showSummary;
+    if (showSummaryProp === true)
+      showSummary = (
+        <Text>
+          Showing {itemsBegin} - {itemsEnd} of {numberItems}
+        </Text>
+      );
+    else showSummary = showSummaryProp;
+
+    // only apply these if the caller hasn't already specified their own theming
+    // otherwise, let the caller control
+    const showSummaryStyles =
+      showSummary && !theme.pagination.container
+        ? {
+            align: 'center',
+            fill: 'horizontal',
+            direction: 'row',
+            justify: 'between',
```

Comment:
Why would we not set these as defaults on `theme.pagination.container`?

Reply:
If we set them as defaults and `summary` is false, the pagination component will justify to the start since there is not another element to force it to the end. We would set them as defaults if we could create a condition:
```
justify: summary ? 'between' : 'end'
```

Reply:
☝️ This was more in line with my thinking. I was thinking that that justify logic could be applied in the `justify` prop of the Container.